schema/schema.go

```diff
@@ -101,6 +102,10 @@ func (ta *Table) AddColumn(name string, columnType string, extra string) {
 		ta.Columns[index].Type = TYPE_STRING
 	}
 
+	if strings.Contains(columnType, "unsigned") || strings.Contains(columnType, "zerofill") {
```

Comment:
According to [this page](https://dev.mysql.com/doc/refman/5.7/en/numeric-type-overview.html):

> If you specify ZEROFILL for a numeric column, MySQL automatically adds the UNSIGNED attribute to the column. 

I'm just covering my bases here.